src/services/user/editDashboardVars.ts

```diff
@@ -0,0 +1,46 @@
+import { AES, enc } from 'crypto-js';
+import { Settings, ISettings, IVariables } from 'db-models-nc';
+import { CustomError, constants } from '../../helpers';
+import config from '../../config';
+
+const { ENCRYPTION_SECRET_KEY } = config.server;
+
+type IEditDashboardVars = (_: IVariables) => Promise<IVariables>
+
+const editDashboardVars: IEditDashboardVars = async ({
+  encryptedVariables,
+  regularVariables,
+}) => {
+  const oldVars = (await Settings.findOne({ where: { name: 'variables' } })) as ISettings;
+  const oldVarsValue = JSON.parse(oldVars.value as string) as IVariables;
+
```

Comment:
No need to parse it as its type JSON in DataTypes so Sequelize will handle parsing it
```suggestion
  const oldVarsValue = oldVars.value as IVariables;
```